src/controller/CHIPDeviceController.cpp

```diff
@@ -1614,6 +1614,11 @@ void OnBasicFailure(void * context, CHIP_ERROR error)
     commissioner->CommissioningStageComplete(error);
 }
 
+void NonConcurrentTimeout(void * context, CHIP_ERROR error)
+{
+    ChipLogProgress(Controller, "Non-Concurrent Mode: Expected NetworkResponse Timeout, do nothing");
```

Comment:
Shouldn't this check that the error was in fact a timeout?

Reply:
Check added